coda-prototype/src/ui/initUI.js

```diff
@@ -561,23 +561,79 @@ function initUI(dataset) {
          * Notifies the user that parsing the dataset file failed, via the alert banner.
          * Prints the first 100 parse errors to the console.
          */
-        function handleDatasetParseError(parseErrors) {
-            UIUtils.displayAlertAsError("Something is wrong with the data format. " +
-                "Change a few things up, refresh and try again.");
+        function handleDatasetParseError(error) {
+            switch (error.name) {
+                case "ParseError":
+                    UIUtils.displayAlertAsError("Something is wrong with the data format. " +
```

Comment:
This is a really annoying message, I know it's a legacy but can we get rid of it?

Consider "Sorry, something is wrong with the data format, Coda can't load it. If you think this is an error. please contact the developers with a copy of the data you're trying to load"